diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2c06f6e..ff85cc9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -31,7 +31,7 @@ endif()
 include(GNUInstallDirs)
 
 find_package(PkgConfig REQUIRED)
-find_package(Qt6 REQUIRED COMPONENTS Core Quick ShaderTools)
+find_package(Qt6 REQUIRED COMPONENTS Core Quick ShaderTools GuiPrivate QuickPrivate)
 qt_standard_project_setup()
 if(QT_KNOWN_POLICY_QTP0004)
     qt_policy(SET QTP0004 NEW)
diff --git a/qml_material/src/kirigami/toolbarlayout.cpp b/qml_material/src/kirigami/toolbarlayout.cpp
index b9c6f76..e8c070f 100644
--- a/qml_material/src/kirigami/toolbarlayout.cpp
+++ b/qml_material/src/kirigami/toolbarlayout.cpp
@@ -8,6 +8,7 @@
 #include <unordered_map>
 
 #include <QDeadlineTimer>
+#include <QElapsedTimer>
 #include <QQmlComponent>
 #include <QTimer>
